great_expectations/expectations/metrics/util.py

```diff
@@ -1058,3 +1059,83 @@ def get_sqlalchemy_source_table_and_schema(
         )
     else:
         return engine.batch_manager.active_batch_data.selectable
+
+
+def get_unexpected_indices_for_multiple_pandas_named_indices(
+    df: pd.DataFrame,
```

Comment:
This is a dataframe of only unexpected results? It looks like it since you do `unexpected_indices: List[Union[int, str]] = list(df.index)`. I would name this something that conveys this instead of `df`. Maybe `unexpected_df`?

Reply:
good point

Reply:
Done :)